curvefs/proto/metaserver.proto

```diff
@@ -206,6 +208,7 @@ message Inode {
     optional uint32 openmpcount = 20; // openmpcount mount points had the file open
     map<string, string> xattr = 21;
     repeated uint64 parent = 22;
+    optional bool streaming = 23;
```

Comment:
The streaming is not a attribute of Inode and I think put it in getInodeResponse is better.

Reply:
> The streaming is not a attribute of Inode and I think put it in getInodeResponse is better.

fixed.